addons/website_slides/__manifest__.py

```diff
@@ -153,7 +153,6 @@
             'web/static/src/session.js',
             'web/static/src/legacy/js/core/cookie_utils.js',
 
-            'web/static/lib/moment/moment.js',
```

Comment:
If you're removing this from the manifest shouldn't you also remove the file? But this should probably stay until we remove moment itself

Reply:
indeed